pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "sparc.client"
-version = "0.0.1.post1"
+version = "0.0.2"
```

Comment:
Can't help but feel that the version should start at `0.1.0`.  If we are following semantic versioning then this proposed change is not a bug fix.

Reply:
Fine with me. 
I used to do bug fixes with .postN releases.

Reply:
I don't have any particular view on which versioning schema. There are different approaches (e.g. 
https://peps.python.org/pep-0440/).
We can agree on using semantic versioning (MAJOR.MINOR.PATCH) if you recommend it @hsorby .

Reply:
I personally prefer semantic versioning because it tells me what has changed between versions. But I don't mind other schemes.  What I would like is for us to declare what versioning scheme we are following.  Even if it is the *do whatever we want* scheme.

Reply:
Let's then use semantic versioning.